modules/get-started/pages/config-rpk-profile.adoc

```diff
@@ -64,16 +64,18 @@ rpk profile use <name-of-profile-to-use>
 
 === Set or edit configuration values
 
+You can customize settings for a single profile. To set a profile's configuration:
+
+* Use xref:reference:rpk/rpk-profile/rpk-profile-set.adoc[`rpk profile set`] to set `key=value` pairs of configuration options to write to the profile's section of `rpk.yaml`.
+* Use xref:reference:rpk/rpk-profile/rpk-profile-edit.adoc[`rpk profile edit`] to edit the profile's section of the `rpk.yaml` file in your default editor.
+
 You can configure settings that apply to all profiles. To set these `globals`:
 
 * Use xref:reference:rpk/rpk-profile/rpk-profile-set-globals.adoc[`rpk profile set-globals`] to set `key=value` pairs to write to the globals section
 of `rpk.yaml`.
 * Use xref:reference:rpk/rpk-profile/rpk-profile-edit-globals.adoc[`rpk profile edit-globals`] to edit the globals section of the `rpk.yaml` file in your default editor.
 
-You can customize settings for a single profile. To set a profile's configuration:
-
-* Use xref:reference:rpk/rpk-profile/rpk-profile-set.adoc[`rpk profile set`] to set `key=value` pairs of configuration options to write to the profile's section of `rpk.yaml`.
-* Use xref:reference:rpk/rpk-profile/rpk-profile-edit.adoc[`rpk profile edit`] to edit the profile's section of the `rpk.yaml` file in your default editor.
+For a list of all the available properties that can be set in your profile, see xref:reference:rpk/rpk-x-options.adoc[`rpk -X options`]
```

Comment:
I swapped the order here as setting configuration values for single profiles are more important than setting globals.